tests/ProductionFilterTest.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+namespace RonasIT\TelescopeExtension\Tests;
+
+use Laravel\Telescope\EntryType;
+use Laravel\Telescope\IncomingEntry;
+use Laravel\Telescope\Watchers\RequestWatcher;
+use RonasIT\TelescopeExtension\Filters\ProductionFilter;
+use Mockery;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProductionFilterTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        config(['telescope.watchers.' . RequestWatcher::class => []]);
+    }
+
+    public function testDevEnv()
+    {
+        $this->mockEnvironment('development');
+
+        $filter = new ProductionFilter();
+
+        $closure = $filter->apply();
+
+        $this->assertTrue($closure(new IncomingEntry([])));
+    }
+
+    public function testLocalEnv()
+    {
+        $this->mockEnvironment('local');
+
+        $filter = new ProductionFilter();
+
+        $closure = $filter->apply();
+
+        $this->assertTrue($closure(new IncomingEntry([])));
+    }
+
+    public function testExceptionProdEnv()
+    {
+        $this->mockEnvironment('production');
+
+        $filter = new ProductionFilter();
+
+        $entry = Mockery::mock(IncomingEntry::class);
+        $entry->type = EntryType::EXCEPTION;
+
+        $entry->shouldReceive('isRequest')->andReturnFalse();
+        $entry->shouldReceive('isClientRequest')->andReturnFalse();
+        $entry->shouldReceive('isSlowQuery')->andReturnFalse();
+        $entry->shouldReceive('isScheduledTask')->andReturnFalse();
+        $entry->shouldReceive('hasMonitoredTag')->andReturnFalse();
+
+        $closure = $filter->apply();
+
+        $this->assertTrue($closure($entry));
+    }
+
+    public function testSuccessRequestProdEnv()
+    {
+        $this->mockEnvironment('production');
+
+        $filter = new ProductionFilter();
+
+        $entry = Mockery::mock(IncomingEntry::class);
+
+        $entry->shouldReceive('isRequest')->andReturnTrue();
+        $entry->shouldReceive('isClientRequest')->andReturnFalse();
+        $entry->shouldReceive('isSlowQuery')->andReturnFalse();
+        $entry->shouldReceive('isScheduledTask')->andReturnFalse();
+        $entry->shouldReceive('hasMonitoredTag')->andReturnFalse();
+
+        $closure = $filter->apply();
```

Comment:
could we make somethof like this? and in other cases as well?

```suggestion
        $entry = new IncomingEntry(['content' => []]);
        $entry->type('request')

        $closure = $filter->apply();
```